app/styles/wizard.less

```diff
@@ -70,7 +70,7 @@
                 }
             }
 
-            a { outline: 0; }
+            a { outline: 0; cursor: pointer; }
```

Comment:
An anchor without an href isn't getting a pointer. Which leads me to think I should probably change the element to a span.